cfripper/__version__.py

```diff
@@ -1,3 +1,3 @@
-VERSION = (0, 12, 2)
+VERSION = (0, 12, 3)
```

Comment:
We need a bigger bump for this. Some of the changes are backwards incompatible (removing-renaming classes).
It's important that we always document backward incompatibilities in the changelog

Reply:
Good points! Now changed to `(0, 13, 0)`.